lib/skc_codegen/src/lambda.rs

```diff
@@ -325,6 +326,7 @@ impl<'hir: 'ictx, 'run, 'ictx: 'run> CodeGen<'hir, 'run, 'ictx> {
                 cond_expr,
                 then_exprs,
                 else_exprs,
+                lvars: _,
```

Comment:
[FYI] you can use `..` when you don't need the value.

Reply:
I thought I would have needed to use the lvars in this match but I forget to use ```..``` 
I will fix that, Thank you